src/shared/services/configuration/map-options.js

```diff
@@ -0,0 +1,20 @@
+import { getCrsRd } from '@datapunt/amsterdam-react-maps/lib/utils';
+
+export const DEFAULT_MARKER_POSITION = {
```

Comment:
I can't think of a situation where we would have a need for a default marker position.

Reply:
I used this to show the defautl marker in this PR, Will remove in the next pr when the click functionality is implemented